test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 name=<aws-iam-user>"
```

Comment:
Does this really have to be `<aws-iam-user>`, or it just an ID that I should pick different from what other users in the same account and region do?

Reply:
Yes. it just needs to be a unique ID. I have already highlighted this in the instructions. See the sentence below: 

``
Use the flag --vpc-name to specify the name for the AWS virtual private cloud to deploy your infrastructure in a logically isolated virtual network: We highly advise utilizing a unique name or the AWS IAM user ID as the identifier for your VPC
``
or simplified version: 

... **We highly advise utilizing a unique name or the AWS IAM user ID as the identifier for your VPC.**